conanfile.py

```diff
@@ -29,10 +29,12 @@ class LibcosimConan(ConanFile):
 
     options = {
         "shared": [True, False],
-        "proxyfmu": [True, False]}
+        "proxyfmu": [True, False],
+        "fmi_log": [True, False]}
```

Comment:
Maybe call this `fmi_logging` instead?

Reply:
changed to `no_fmi_logging` with a default `False`